### PR TITLE
fix:  Create pages from markdown files #12

### PR DESCRIPTION
**Pull request by olavea**
Can I keep my comments in gatsby-node.js for my story Queen @raae ?

### CODE REVIEW

**Comment by raae** [comment]
> Can I keep my comments in gatsby-node.js for my story Queen @raae ?

I would rather not, it's very confusing for those who are not initiated and so is the naming.

File: gatsby-node.js [hunk 1]
```diff
@@ -4,31 +4,29 @@
 //                 Baking pages
 //                 with Cap'n Granny Sharksby's
 //                 createPages hook
-async function bakeGingerNodesIntoGoodies({ graphql, actions }) {
-  console.log("Captain Granny Sharksbys createPages hook💀");
+async function bakeMarkdownNodesIntoGoodies({ graphql, actions }) {
```
Comment (raae) [inline gatsby-node.js:7]: You are making pages, not goodies. Please rename to `bakeMarkdownNodesIntoPages`.

File: gatsby-node.js [hunk 2]
```diff
@@ -4,31 +4,29 @@
...
+  // console.log("Captain Granny Sharksbys createPages hook💀");
   //              1. bakingSong = Troya Goatsby and Lizabeth Catsby sings
   //                 the bakingSong by Cap'n Granny Sharksby 🦢
   const bakingSong = require.resolve("./src/templates/bakingSong.js");
```
Comment (raae) [inline gatsby-node.js:11]: Please rename `pageTemplate`.

File: gatsby-node.js [hunk 3]
```diff
@@ -53,50 +51,5 @@ async function bakeGingerNodesIntoGoodies({ graphql, actions }) {
 exports.createPages = async (params) => {
```
Comment (raae) [inline gatsby-node.js:51]: Rename params `gatsbyUtils`

File: gatsby-node.js [hunk 4]
```diff
@@ -4,31 +4,29 @@
...
+      supplies: allMarkdownRemark {
         nodes {
           id
         }
       }
     }
   `);
   //             3. Loop over the nodes and for each create a page
-
   data.supplies.nodes.forEach((ahoyGoodie) => {
```
Comment (raae) [inline gatsby-node.js:23]: You may keep supplies, but please rename ahoyGoodie to `node`.

**Review by raae — CHANGES_REQUESTED** [review]

File: gatsby-node.js (hunk 1 above)
Comment (olavea) [reply gatsby-node.js]: 😺 👍

File: gatsby-node.js (hunk 2 above)
Comment (olavea) [reply gatsby-node.js]: 😺 👍

File: gatsby-node.js (hunk 3 above)
Comment (olavea) [reply gatsby-node.js]: 😺 👍

File: gatsby-node.js (hunk 4 above)
Comment (olavea) [reply gatsby-node.js]: 😺 👍

**Comment by olavea** [comment]
😺 👍

**Pull request merged**